ICSharpCode.CodeConverter/CSharp/DocumentExtensions.cs

```diff
@@ -21,7 +21,11 @@ public static async Task<Document> WithSimplifiedSyntaxRootAsync(this Document d
         {
             var root = syntaxRoot  ?? await doc.GetSyntaxRootAsync();
             var withSyntaxRoot = doc.WithSyntaxRoot(root.WithAdditionalAnnotations(Simplifier.Annotation));
-            return await Simplifier.ReduceAsync(withSyntaxRoot);
+            try {
+                return await Simplifier.ReduceAsync(withSyntaxRoot);
+            } catch {
+                return doc;
+            }
```

Comment:
This probably means some of our returned syntax trees are malformed. The only way we'll find out is if people report it as a bug that code isn't as simple as it could be. I'll merge as is, but it'd be good to get this logged to the output window via iprogress or something